docs/lib/info/fragments/ios/app-uninstall.md

```diff
@@ -0,0 +1,5 @@
+Some Amplify categories such as Analytics, Auth, and DataStore persist data to the local device. Some of that data is automatically removed when a user uninstalls the app from the device.
```

Comment:
Should we also mention what data get lost when the app is uninstalled in this doc?

Reply:
I thought about it, but in the absence of a full enumeration of data that falls into that category, felt it was more useful to simply explicitly call out the data that is *not* erased.